### PR TITLE
Fixes hero/banner spacing 

### PR DESCRIPTION
**Pull request by isabelsang**
Banner (aka hero) now extends to the width of the full page.  Text is now centered (although it is not centered vertically, but I'm sure I can change that if people think it would look better).  I think I want to change the background photo, let me know if you have any suggestions.  

In order to have the banner extend, I commented out a line of code in the content class in Dashboard but if this causes issues on other pages, we can figure out another solution.  

ps I have been trying to figure out why people call it a "hero" but haven't had much luck... the mystery remains

image

### CODE REVIEW

**Comment by isabelsang** [comment]
I cropped the photo so I like it a little better now but still don't know if I love it... any thoughts? (I won't be offended at all if you think I should switch out the photo entirely)
image

File: frontend/src/components/dashboard/Banner.js [hunk 1]
```diff
@@ -5,36 +5,43 @@ import PropTypes from 'prop-types';
...
-    backgroundRepeat: 'no-repeat',
-    backgroundSize: 'cover',
-    display: 'flex',
-    alignItems: 'center',
-    justifyContent: 'center',
-    // height: '300px',
+    height: '40vh',
+    width: '100%',
     position: 'relative',
+    marginBottom: theme.spacing(2),
```
Comment (katch00) [inline frontend/src/components/dashboard/Banner.js:11]: nit: should be in alphabetical order, same for bannerText

**Comment by isabelsang** [comment]
Just added a really small commit to this PR.  I centered the hero text and made the description text below the header larger.  
image

**Comment by indhu-gunda** [comment]
Actually it did mess with padding on the close-up view: 
[https://screenshot.googleplex.com/c1zSwGXOj1c.png](url)

**Comment by indhu-gunda** [comment]
An option could be to set the css class based on a variable like this does for when the drawer is open/closes (In Dashboard.js): 

```
        <main
          className={clsx(classes.content, {
            [classes.contentShift]: open,
          })}
        >
```

**Pull request merged**